src/chrome/content/rules/Statushub.io.xml

```diff
@@ -0,0 +1,20 @@
+<!--blog. protocol error
+mozillastatus. mismatch
```

Comment:
customer get their own subdomain, which are not securable.
can you document this behavior instead of just list them?

Note that you need to keep the separate comment for the blog. subdomain.